bedrock/firefox/templates/firefox/whatsnew/whatsnew-56.html

```diff
@@ -0,0 +1,63 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base-pebbles.html" %}
```

Comment:
Please add a simple lang file reference to the template

```
{% add_lang_files "firefox/whatsnew_56" %}
```

Depending on the answer (German only?), we might need to add it to the base template too.

Reply:
Never mind the last part, this template has its own title and desc